### PR TITLE
* draft of the abstract

### PR DESCRIPTION
**Pull request by esc**
As title.

### CODE REVIEW

File: abstract.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+a cacophony of largely unprincipled hacks, workarounds and heuristics in order
+to support new Python versions. In this talk I will introduce a new,
+principled approach to regularize (or sanitize) Python bytecode. The ideas used
+here are inspired by recent publications in theoretical computer science on the
+topics of Regionalized State Value Dependency Graphs or RVSDGs as they are
+commonly known. While we don't transfer directly to RVSDGs, we have found that
+an intermediary representation introduced in (ref:: paper) may be sufficient for
+Numba's needs. The basic idea is to take the bytecode spaghetti, which
+effectively describes a Control Flow Graph (CFG) of Basic Blocks (BBs) and
+transform this into a sane format. Some features of this format are the clear
```
Comment (sklam) [inline abstract.md]: "sane" 😏 
maybe "structured" for the final version

**Review by sklam — COMMENTED** [review]
Nice! slight worry if we can get to talk about JAX and PyTorch by then.

File: abstract.md (hunk 1 above)
Comment (esc) [reply abstract.md]: I have submitted a second draft, where I re-wrote several things.

File: abstract.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Tree (AST). This technique may enable opportunities to open multiple gateways
+into the future and will provide us with ample possibilities for sustained
+research and innovation.
+
+Furthermore, this approach is part of a larger effort to transform the Numba
+codebase into a compiler building toolkit that will become a framework to allow
+for the creation of highly specialised compilers for the dawning era of diverse
+and heterogeneous silicon that our civilisation is venturing into. I will
+conclude the talk with an outlook of potential innovations in compiler
+technologies during the next decade and beyond!
```
Comment (sklam) [inline abstract.md]: would it be better to conclude with how the new compiler tech can transform scientific python so it ties back to the conference more?

Comment (esc) [reply abstract.md]: yes, I tied it back to scientific computing and beyond multi-core chips in the last paragraph now, I think it rounds it off nicely.

**Comment by esc** [comment]
Thank you @sklam @stuartarchibald and @leriomaggio for your feedback, both on GH and OOB.

**Pull request merged**